File: lua/axr.lua
-- Copyright 2026 drblez AKA Ruslan Stepanenko (drblez@gmail.com)
--
-- Lua/TSO AXR module stubs.
--
-- Object Table:
-- | Object | Kind | Purpose |
-- |--------|------|---------|
-- | axr.request | function | AXR request interface (optional) |
local axr = {}

function axr.request(_)
  error("LUZ-13001 axr.request not implemented")
end

return axr

File: src/axr.c
/*
 * Copyright 2026 drblez AKA Ruslan Stepanenko (drblez@gmail.com)
 *
 * Lua/TSO AXR API stubs.
 *
 * Object Table:
 * | Object | Kind | Purpose |
 * |--------|------|---------|
 * | lua_axr_request | function | Execute AXR request (optional) |
 */
#include "axr.h"
#include "errors.h"

int lua_axr_request(const char *request)
{
  (void)request;
  return LUZ_E_AXR_REQUEST;
}
